runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotConfig.java

```diff
@@ -121,14 +121,14 @@ default boolean uploadScreenshot()
 	}
 
 	@ConfigItem(
-		keyName = "hotkey",
-		name = "Screenshot hotkey",
-		description = "When you press this key a screenshot will be taken",
-		position = 10
+			keyName = "boss",
```

Comment:
Don't rearrange this just to insert boss kills; it is probably fine to be after the PvP kills entry.